### DISCUSSION TITLE
support binary build for EL9 or CENTOS 9 Stream

### DISCUSSION THREAD

**Discussion by TomasKulhanek** [OpenModelica/OpenModelica#9398] [Ideas]
Hi, builds for RPM based Linux distros are supported for Enterprise Linux (EL or CENTOS) 6,7,8 and Fedora 25-36. However nothing from previous builds seems to be usable in RHEL 9 or CENTOS 9 Stream. A build for RHEL 9 or CENTOS 9 would be very appreciated.

**Comment by sjoelund** [comment]
I have started a build using Almalinux 9 at https://test.openmodelica.org/jenkins/blue/organizations/jenkins/LINUX_BUILDS/detail/LINUX_BUILDS/1875/pipeline

**Comment by TomasKulhanek** [comment]
Great. Thank you. Current build for el9-nightly and el9-1.19 seems failing on Permission denied:
```
Failed SFTP PUT: /var/lib/jenkins/ws/LINUX_BUILDS/rpmbuild/RPMS/x86_64/openmodelica-nightly-
1.20.0~dev~295~g02c4db5-1.el9.x86_64.rpm -> build:/var/www/rpm/el9/: (SSH_FX_PERMISSION_D
ENIED: The user does not have sufficient permissions to perform the operation): Permission denied

1.25.7  · Core 2.346.3  · d1a41d
```

**Reply by sjoelund** [reply]
Now it should be working. At least the build finished and a repository was generated

**Comment by casella** [comment]
Nice that we got at least one Blue Ocean pipeline report with all green marks: https://test.openmodelica.org/jenkins/blue/organizations/jenkins/LINUX_BUILDS/detail/LINUX_BUILDS/1878/pipeline

The next one had one broken again https://test.openmodelica.org/jenkins/blue/organizations/jenkins/LINUX_BUILDS/detail/LINUX_BUILDS/1879/pipeline, but this time it is fc35, so that's another story 😅

**Comment by TomasKulhanek** [comment]
Thanks. I have scripts to create VM https://github.com/creative-connections/Bodylight-VirtualMachine from scratch using Centos 9 stream. Now OpenModelica can be installed there from binary builds. Thank you.
